### PR TITLE
Distinguish between Zendesk feedback on different environments

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

File: cla_public/apps/base/forms.py [hunk 1]
```diff
@@ -49,15 +51,21 @@ class FeedbackForm(Honeypot, BabelTranslationsFormMixin, Form):
         validators=[InputRequired()])
 
     def api_payload(self):
+        environment = os.environ.get('CLA_ENV')
```
Comment (s-block) [inline cla_public/apps/base/forms.py]: could add this to the main settings file

CLA_ENV = os.environ.get('CLA_ENV')

then

environment = current_app.config['CLA_ENV']

**Comment by s-block** [comment]
Looks good

**Comment by tyom** [comment]
Updated.

**Comment by s-block** [comment]
Looks good

**Pull request merged**